src/Reflection/ReflectionClass.php

```diff
@@ -168,8 +168,7 @@ public static function createFromNode(
     public function getShortName() : string
     {
         if (! $this->isAnonymous()) {
-            /** @var Node\Identifier $this->node->name */
-            return $this->node->name->name;
+            return (string)$this->node->name;
```

Comment:
Can you make this a separate PR for this change please? Also what does this change benefit us by?